tests/src/daw_json_link_test.cpp

```diff
@@ -32,7 +32,7 @@
 Temporarily disable the constexpr tests in MSVC when C++20
 */
 #if defined( _MSC_VER ) and not defined( __clang__ ) and \
-  defined( DAW_CXX_STANDARD ) and DAW_CXX_STANDARD == 20
+  __cplusplus > 201703L
```

Comment:
This breaks on MSVC as the version is locked at the 98 release.  These all need to be buildable without cmake/vcpkg.  This is working around a compiler bug in MSVC when /std:c++latest is enabled which enables their C++20 code paths.

Reply:
https://docs.microsoft.com/en-us/cpp/build/reference/zc-cplusplus

This flag is passed in the `flags-windows` presets.